lib/ui/ui_dart_state.cc

```diff
@@ -114,7 +115,10 @@ fml::WeakPtr<SnapshotDelegate> UIDartState::GetSnapshotDelegate() const {
 }
 
 fml::WeakPtr<GrContext> UIDartState::GetResourceContext() const {
-  return resource_context_;
+  if (!io_manager_) {
+    return fml::WeakPtr<GrContext>();
```

Comment:
Nit: Since the type is already known, you can replace this line with just `return {};`

Reply:
Done